R/installspec.r

```diff
@@ -14,11 +20,11 @@ installspec <- function(user = TRUE) {
             break
         }
     }
-    
+
     if (!found_binary)
-        stop('Jupyter or IPython 3.0 has to be installed but could neither run “jupyter” nor “ipython”, “ipython2” or “ipython3”.
-             (Note that “ipython2” is just IPython for Python 2, but still may be IPython 3.0)')
-    
+        stop('Jupyter or IPython 3.0 has to be installed but could neither run "jupyter" nor "ipython", "ipython2" or "ipython3".
+             (Note that "ipython2" is just IPython for Python 2, but still may be IPython 3.0)')
```

Comment:
oh, damn, too late. sorry.

but i [don’t think it’s that bad](https://github.com/IRkernel/IRkernel/blob/bcaea6800a76c76740d6f01818d302674aa73da6/R/installspec.r#L25-L27).

fucking R/Windows not getting their collective shit together